src/swupd_curl.h

```diff
@@ -45,6 +46,11 @@ typedef bool (*swupd_curl_error_cb)(enum download_status status, void *data);
  */
 typedef void (*swupd_curl_free_cb)(void *data);
 
+/*
+ * Callback called periodically by curl to report on download progress
+ */
+typedef int (*swupd_curl_progress_cb)(void *clientp, int64_t dltotal, int64_t dlnow, int64_t UNUSED_PARAM ultotal, int64_t UNUSED_PARAM ulnow);
```

Comment:
Remove the UNUSED_PARAM from declarations

Reply:
done